src/components/WorkHistory/index.tsx

```diff
@@ -8,21 +8,37 @@ import debounce from 'lodash/debounce';
 import { ReactComponent as DeleteIcon } from 'assets/illustrations/icons/delete.svg';
 import months from 'utils/getMonthsArray';
 import numberRange from 'utils/numberRange';
+import CreatableSelect from 'react-select/creatable';
+import { Link } from 'react-router-dom';
+import creatableReactSelectStyles from './createSelectStyles';
 import {
   useAddProfileWorkplace,
   useRemoveProfileWorkplace,
   useUpdateProfileWorkplace,
+  useFetchAllOrganisations,
 } from './hooks';
 import styles from './index.module.scss';
 
 const WorkHistory: FC = () => {
+  const fetchAllOrganisations = useFetchAllOrganisations();
+
   const workplaces = useSelector(
     (state: RootState) => state.profile.workplaces
   );
+
   const isEditable = useSelector(
     (state: RootState) => state.profile.isEditable
   );
 
+  const workplaceOrganisation = (name: string) =>
+    fetchAllOrganisations &&
+    fetchAllOrganisations.normalizedOrganisationsData &&
+    fetchAllOrganisations.normalizedOrganisationsData.filter(
+      function (organisation: { label: string }) {
+        return organisation.label === name;
+      }
+    );
```

Comment:
Move this to `utils` folder. And it should not have any side effect.
I should pass **x**, **y**, ... to the function and it should return me **z**
Shouldn't be unpredictable.

I pass data and name and it should return me an array based on that.